rtmp_to_hls/H264_codec.md

```diff
@@ -0,0 +1,94 @@
+In this chapter you will learn about the H264 codec and how its processing is done with the Membrane Framework plugin - the [H264 plugin](https://github.com/membraneframework/membrane_h264_ffmpeg_plugin).
+
+In H264 we can distinguish two layers of abstraction - the video coding layer (VCL), focused on the visual representation of the video, and the network abstraction layer (NAL), focused on how to structurize the stream so that it can be sent via the network.
+
+When it comes to VCL, a video encoded with the H264 codec is represented as a sequence of pictures. Each picture consists of many **macroblocks**.
```

Comment:
I think you may add a picture here illustrating macroblock, e.g. the one from the confluence page: https://membraneframework.atlassian.net/wiki/spaces/MEMBRANEFR/pages/16613377/H264+Parser .

Reply:
Good idea!

Reply:
Okay